tests/llm/utils/mock_dal.py

```
@@ -2,26 +2,34 @@
     def __init__(
         self,
         test_case_folder: Path,
         issue_data: Optional[Dict],
+        issues_metadata: Optional[List[Dict]],
         resource_instructions: Optional[ResourceInstructions],
         generate_mocks: bool,
     ):
-        super().__init__(cluster="test")
+        try:
+            super().__init__(cluster="test")
+        except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)